### ISSUE TITLE
Feature: post comment when previously-denied user accepts CLAs

### ISSUE THREAD

**Issue by dfawley** [communitybridge/easycla#2279]
This is issue #310, which was closed as "done", but not done, and got no response after pinging.  Please just close this and reopen that.  However, I'll put the FR here just in case:

---

The focus of #269 was changed after it was created.  The new version was implemented and alleviated some of the problems of managing CLA issues, but not all of them.  Hence, I am reposting the original feature request in a new issue.

---

**Summary**

Post comment when previously-denied user accepts CLAs.

**Background**

If a contributor has not accepted the CLAs when creating a PR in a managed repo, the CLA bot will post a message indicating this.  If the user then accepts the CLAs, this comment will be changed to indicate they have accepted the CLAs.  However, if I, as a maintainer, was waiting for this action in order to merge the PR, no update will be sent when this happens, and I will have to poll the PR to find out it's ready to merge.

The mention in [this comment](https://github.com/grpc/grpc-go/pull/3084#issuecomment-540091788) that grpc maintainers didn't want an update for a user that has accepted the CLAs was in regards to users that were *already* OK when the PR was created.  We still wouldn't want messages in these cases.  Only after the bot has posted a comment indicating the CLAs were denied would we want an update.

**User Story**

As a maintainer, I would like a new comment to be posted when a previously-denied contributor accepts the CLAs in order to avoid the need to poll the PR to discover its state.  Note that this is in-line with how the previous CNCF CLA bot worked 

cc @jpalmerLinuxFoundation
cc @dealako

**Comment by dealako** [comment]
Won't fix. Out of scope of current product requirements/features.

**Comment by dfawley** [comment]
This is a usability concern, and one that hasn't faded in the 1.5+ years since I created this issue.  Why is "usability" considered "out of scope"?
